distribution/ddf-common/src/main/resources/security/configurations.policy

```diff
@@ -42,3 +42,26 @@ grant codeBase "file:/catalog-core-directorymonitor/org.apache.camel.camel-core/
     // permission java.io.FilePermission "/test/some/monitored/directory", "read";
     // permission java.io.FilePermission "/test/some/monitored/directory${/}-", "read, write";
 }
+
+//                        URL Resource Reader Permissions
+// Adding required permissions for the URL Resource Reader to retrieve file resources.
+//
+//
+// Configuring URL Resource Reader requires adding read and write permissions to the directory being monitored.
```

Comment:
✏️ Configuring URL Resource Reader requires adding read permissions to the directory being read from.

Reply:
fixed